StreetDrop/StreetDrop/Presentation/Settings/ViewModel/SettingsViewModel.swift

```diff
@@ -62,23 +62,87 @@ private extension DefaultSettingsViewModel {
             .subscribe { savedMusicAppInServer in
                 output.savedMusicAppInServer.accept(savedMusicAppInServer)
             } onFailure: { error in
-                print(error.localizedDescription)
+                output.changingMusicAppFailAlert.accept("연결 앱 변경이 실패했어요!")
             }
             .disposed(by: disposeBag)
     }
+}
+
+private extension DefaultSettingsViewModel {
+    func fetchSettingSectionTypes(output: Output, disposeBag: DisposeBag) {
+        Observable.zip(
+            fetchDefaultSettingSectionTypes(),
+            checkHasNewNotice(),
+            fetchMyMusicApp(output: output)
+        )
+        .map { settingSectionTypes, hasNewNotice, myMusicApp in
+            self.updateSettingSectionTypes(
+                settingSectionTypes,
+                hasNewNotice: hasNewNotice,
+                myMusicApp: myMusicApp
+            )
+        }
+        .subscribe(onNext: { updatedSettingSectionTypes in
+            output.defaultSettingSectionTypes.accept(updatedSettingSectionTypes)
+        })
+        .disposed(by: disposeBag)
+    }
+    
+    func fetchDefaultSettingSectionTypes() -> Observable<[SettingSectionType]> {
+        Observable.create { observer in
+            let settingSectionTypes = self.useCase.fetchDefaultSettingSectionTypes()
+            observer.onNext(settingSectionTypes)
+            observer.onCompleted()
+            return Disposables.create()
+        }
+    }
+    
+    func checkHasNewNotice() -> Observable<Bool> {
+        self.useCase.fetchLastSeenNoticeId()
+            .flatMap { [weak self] noticeId -> Single<Bool> in
+                guard let self else { return Single.just(false) }
+                return self.useCase.checkNewNotice(lastNoticeId: noticeId)
+            }
+            .asObservable()
+    }
     
-    func fetchMymusicAppFromLocal(output: Output, disposeBag: DisposeBag) {
+    private func fetchMyMusicApp(output: Output) -> Observable<MusicApp> {
```

Comment:
해당 메서드 extension private 입니다!

```suggestion
    func fetchMyMusicApp(output: Output) -> Observable<MusicApp> {
```